### PR TITLE
Feature/InHeroLabs flag

### PR DESCRIPTION
**Pull request by Surxe**
Adds a flag to hero data for whether or not the hero is exclusive to Hero labs.

Used in [[Template:Heroes count]] on [[Hero Labs]]

### CODE REVIEW

File: src/parser/parsers/heroes.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ def run(self):
                     'IsDisabled': hero_value['m_bDisabled'],
                 }
 
+                # Key is missing from released heroes
+                # Frontend will need to use if "m_bAvailableInHeroLabs or not m_bInDevelopment"
+                if 'm_bAvailableInHeroLabs' in hero_value:
+                    hero_stats['InHeroLabs'] = hero_value['m_bAvailableInHeroLabs']
```
Comment (hassayag) [inline src/parser/parsers/heroes.py:32]: similar to `InDevelopment` flag, can we instead set the flag, but default to False. ie.

```suggestion
                hero_stats['InHeroLabs'] = hero_value.get('m_bAvailableInHeroLabs', False)
```

**Review by hassayag — CHANGES_REQUESTED** [review]
good shout, just 1 smol thing

**Comment by Surxe** [comment]
> good shout, just 1 smol thing

Didn't know there was integration for suggesting code, neat!

**Review by hassayag — APPROVED** [review]
🤌

**Pull request merged**